code/game/objects/items/weapons/grenades/plastic.dm

```diff
@@ -78,6 +102,12 @@
 		src.target = AM
 		forceMove(null)	//Yep
 
+		if(istype(AM, /obj/item)) //your crappy throwing star can't fly so good with a giant brick of c4 on it.
+			var/obj/item/I = AM
```

Comment:
This whole thing could really use some encapsulation
Like Attach() proc for objects or smth liek dis